cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java

```diff
@@ -0,0 +1,16 @@
+package com.authentication.apikey;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AuthenticationApiKeyClientTest {
```

Comment:
I should have added test for auth?

Reply:
```  
"Authentication_ApiKey_valid": "pass",
  "Authentication_ApiKey_invalid": "fail",
  "Authentication_OAuth2_valid": "pass",
  "Authentication_OAuth2_invalid": "fail",
```

Auth tests seems failed, so I'm still working on figuring out the reason

Reply:
ignore for now, the 2 API is design to fail as invalid credential.

Reply:
ok, I checked the logs, the invalid calls are called, maybe `fail` is expected, I will delete the file and ignore those two `fail`

Reply:
said it is expected, the call is designed to return 401/403

please prioritize on those does not have test yet.

Reply:
I'm working on other cases now.